lib/network.py

```diff
@@ -525,7 +525,7 @@ def _add_recent_server(self, server):
 
     def _process_response(self, interface, response, callbacks):
         if self.debug:
-            self.print_error("<--", response)
+            self.print_error(interface.host, "<--", response)
```

Comment:
did you test this? I think it's redundant, it happens already.
see https://github.com/spesmilo/electrum/blob/12c5474cf126f8bcb01a5cf90b582189f1297cb0/lib/util.py#L172